internal/plugin/connectors/tcp/mysql/connection_details_test.go

```diff
@@ -12,14 +12,17 @@ func TestExpectedFields(t *testing.T) {
 		"port":     []byte("1234"),
 		"username": []byte("myusername"),
 		"password": []byte("mypassword"),
+		"sslmode":  []byte("disable"),
 	}
 
 	expectedConnDetails := ConnectionDetails{
-		Host:     "myhost",
+		Host: "myhost",
+		Options: map[string]string{
+			"sslmode": "disable",
```

Comment:
is that actually the intended flow for this test? the default SSL mode is "require" (see [here](https://github.com/cyberark/secretless-broker/blob/2dd84c3d152ff878ed31b5b552dbc8c5d12ee2d8/internal/plugin/connectors/tcp/ssl/ssl.go#L34))

Reply:
leaving it blank would effectively be the same as "require"

Reply:
This tests explicitly sets it to "disable" about 5 lines up from here